tests/test_filtsmooth/test_gaussfiltsmooth/_linearization_test_interface.py

```diff
@@ -0,0 +1,168 @@
+"""Test interface for EKF and UKF."""
```

Comment:
this function is now the pytest version of the former `filtsmooth_testcases`.

Reply:
Ah, yes, thanks for taking care of this!